compose/src/main/java/com/bottlerocketstudios/compose/profile/ProfilePreviewData.kt

```diff
@@ -0,0 +1,11 @@
+package com.bottlerocketstudios.compose.profile
+
+import androidx.compose.runtime.mutableStateOf
+
+val ProfileMockData = ProfileScreenState (
```

Comment:
@VanyaHuamanBottleRocket nit: Mark as `internal` and use camelCase (lowercase the first character)